radis/api/hitempapi.py

```diff
@@ -125,8 +125,8 @@ def get_last(b):
 def setup_credentials():
     """Set up HITRAN credentials and store them in .env file"""
     # Get credentials from user
-    username = input("Enter HITRAN username: ")
-    password = getpass("Enter HITRAN password: ")
+    username = os.environ.get("HITRAN_USERNAME") or input("Enter HITRAN username: ")
+    password = os.environ.get("HITRAN_PASSWORD") or getpass("Enter HITRAN password: ")
```

Comment:
```suggestion
    username = os.environ["HITRAN_USERNAME"] or input("Enter HITRAN username: ")
    password = os.environ["HITRAN_PASSWORD"] or getpass("Enter HITRAN password: ")
```

Reply:
based on https://stackoverflow.com/questions/21047182/how-to-set-environment-variables-in-travis-ci-and-access-them-from-python-script